### PR TITLE
Switching test to kubeflow deployment

### PR DESCRIPTION
**Pull request by IronPan**
The pipeline is heading towards to the Kubeflow kfctl.sh deployment. Switch the way the tests are deploying the cluster so tests are testing against actual user scenario. 

This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/351)

### CODE REVIEW

File: test/presubmit-tests.sh [hunk 1]
```diff
@@ -69,8 +69,8 @@ echo "presubmit test starts"
 
 # activating the service account
 gcloud auth activate-service-account --key-file="${GOOGLE_APPLICATION_CREDENTIALS}"
+gcloud config set compute/zone us-central1-a
```
[the remark this answers is not in the document]
Comment (IronPan) [reply test/presubmit-tests.sh]: currently we only requested quota for this zone.

File: test/presubmit-tests.sh [hunk 2]
```diff
@@ -69,8 +69,8 @@ echo "presubmit test starts"
...
 
-#Creating a new GKE cluster if needed
```
[the remark this answers is not in the document]
Comment (IronPan) [reply test/presubmit-tests.sh]: reverted

**Comment by gaoning777** [comment]
Is the major change to configure the GCP credentials?

File: test/install-argo.sh [hunk 3]
```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```
Comment (Ark-kun) [inline test/install-argo.sh]: bash -e

File: test/presubmit-tests-with-pipeline-deployment.sh [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+TEST_CLUSTER_PREFIX=${WORKFLOW_FILE%.*}
+TEST_CLUSTER=$(echo $TEST_CLUSTER_PREFIX | cut -d _ -f 1)-${PULL_PULL_SHA:0:7}-${RANDOM}
+
+export CLIENT_ID=${RANDOM}
+export CLIENT_SECRET=${RANDOM}
+KFAPP=${TEST_CLUSTER}
+
+function clean_up {
+  echo "Clean up..."
+  cd ${KFAPP}
```
Comment (Ark-kun) [inline test/presubmit-tests-with-pipeline-deployment.sh]: This fails since `$KFAPP` is relative path.

**Pull request merged**